scripts/imgtool/keys/__init__.py

```diff
@@ -53,13 +53,13 @@ def load(path, passwd=None):
                 backend=default_backend())
 
     if isinstance(pk, RSAPrivateKey):
-        if pk.key_size != 2048:
+        if pk.key_size not in [2048, 3072]:
```

Comment:
It looks like there's a variable for this (`RSA_KEY_SIZES`); why not use it in this file?

Reply:
Fixed